packages/clay-empty-state/stories/index.tsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import '@clayui/css/lib/css/atlas.css';
+import ClayButton from '@clayui/button';
+import {storiesOf} from '@storybook/react';
+import React from 'react';
+
+const emptyImage = require('../images/empty_state.gif');
```

Comment:
lets move these files into `clay-empty-state/stories/images` rather than leave at the root level.